packages/office-addin-dev-settings/src/dev-settings-windows.ts

```diff
@@ -17,9 +17,6 @@ const UseDirectDebugger: string = "UseDirectDebugger";
 const UseLiveReload: string = "UseLiveReload";
 const UseProxyDebugger: string = "UseWebDebugger";
 const WebViewSelection: string = "WebViewSelection";
-const WebViewTypeString_IE: string = "IE";
-const WebViewTypeString_Edge: string = "Edge";
-const WebViewTypeString_EdgeChromium: string = "Edge Chromium";
```

Comment:
@akrantz I removed these consts because I have the same strings baked into the enums themselves now.